apis/python/src/tiledbsoma/util.py

```diff
@@ -128,6 +128,10 @@ def slice_to_range(
     if stop is None:
         stop = nonempty_domain[1]
 
+    # Indexing beyond end of array should "trim" to end of array
```

Comment:
Possibly naive question:  do we need the same for `start` coordinate?   I.e., bound it to zero?

Reply:
sure